src/Feature/Editor/EditorConfiguration.re

```diff
@@ -112,6 +113,38 @@ module CustomDecoders: {
       ~encode=Json.Encode.float,
     );
 
+  let fontWeight =
+    custom(
+      ~decode=
+        Json.Decode.(
```

Comment:
One minor improvement here would be to also accept number values, like `"editor.fontWeight": 500`. We could do that by combing a `number` decoder too, like:

```
let fontWeightDecoder = Json.Decode.(
   one_of([
      ("fontWeight.number", number |> map(
           fun
           | 100 => Revery.Font.Weight.Thin
           ...
           | 900 => Revery.Font.Weight.Heavy
           | _ => Revery.Font.Weight.Normal
       )),
       ("fontWeight.number", string |> map(
           fun
           | "100" => Revery.Font.Weight.Thin
           ...
           | "900" => Revery.Font.Weight.Heavy
           | _ => Revery.Font.Weight.Normal
       )),
   ])
);
```

And then using our custom decoder:
```
let fontWeight = 
   custom(
       ~decode=fontWeightDecoder,
       ...
   );
```

Reply:
Sounds good!